kentikapi/client.go

```diff
@@ -83,9 +112,26 @@ func NewClient(c Config) *Client {
 		c.SyntheticsAPIURL = syntheticsAPIURL
 	}
 
+	if c.CloudExportGRPCHostPort == "" {
+		c.CloudExportGRPCHostPort = cloudExportGRPCHostPort
+	}
+
+	if c.SyntheticsGRPCHostPort == "" {
+		c.SyntheticsGRPCHostPort = syntheticsGRPCHostPort
+	}
+
 	cloudexportClient := cloudexport.NewAPIClient(makeCloudExportConfig(c))
 	syntheticsClient := synthetics.NewAPIClient(makeSyntheticsConfig(c))
```

Comment:
Having both `syntheticsClient` and `syntheticsConnection` is pretty confusing. Ultimately there should be 1 `syntheticsClient` support either HTTP or gRPC transport.

Reply:
... and of course the same for any other APIv6 end-point.